### ISSUE TITLE
Ideas for items on the dashboard

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12638]
### Tell us your idea!

This is a collection of feedback/ideas for contents of our new dashboard coming in from beta users over different channels.
Once an item is decided to be worth working on it should be taken into a dedicated issue.

### What solution would you suggest?

List of ideas as they came in (maybe rephrased if is was part of a longer dialog). This is neither rated nor checked on items we might already have dismissed.

- Make the search option (magnifier icon) in the action bar a part of the dashboard content instead. Idea behind: Offer a prominent search as a "jump pad" to quickly jump to e.g. the desired stored cache or into the keyword search
- Have the "Go to" link not hidden in the menu but on the dashboard
- GC-Droid has a flashlight button on its dashboard, would love to see this on your dashboard as well
- When reading the new "xyz caches stored" I thought about adding the amount of lists as well, like "xyz caches stored on abc lists"
- Besides the compass the dashboard is the only place to see coordinates and GPS state (accuracy, etc.). It could be a bit more prominent there to always see if the main input needed for Geocaching is alright.
- You could use the dashboard to link to your FAQ, user manual and donation. Tons of empty space there.
- Please allow setting a wallpaper to that home screen or at least provide some defaults to choose from. Its ugly in poor white...

**Comment by fm-sys** [comment]
> E.g. In #242 it was suggested to add the nearest cache to the dashboard...

But OTOH we should keep in mind that some people have more than one connector enabled.

**Comment by ztNFny** [comment]
> * Make the search option (magnifier icon) in the action bar a part of the dashboard content instead. Idea behind: Offer a prominent search as a "jump pad" to quickly jump to e.g. the desired stored cache or into the keyword search

I'm against this, having search in toolbar is a standard place that many apps use. Moving it to dashboard would make it a moving target, depending on the content that's shown in dashboard, e.g. when there's an announcement, which is horrible for UX.

> * Have the "Go to" link not hidden in the menu but on the dashboard
> * GC-Droid has a flashlight button on its dashboard, would love to see this on your dashboard as well

Wasn't the point of bottom-nav to not have dashboard with buttons? Flashlight should be easily accessible from Androids Quick settings already.

> * When reading the new "xyz caches stored" I thought about adding the amount of lists as well, like "xyz caches stored on abc lists"
> * Besides the compass the dashboard is the only place to see coordinates and GPS state (accuracy, etc.). It could be a bit more prominent there to always see if the main input needed for Geocaching is alright.
> * You could use the dashboard to link to your FAQ, user manual and donation. Tons of empty space there.

Sounds ok to me, even though personally I don't need any of them.

**Comment by Lineflyer** [comment]
> But OTOH we should keep in mind that some people have more than one connector enabled.

Doesn't matter to much as any of these is still the closest.

**Comment by wolverine007** [comment]
Not sure if it is technically possible, but maybe a user defined layout?

So the user could select what is shown on the dashboard and could for instance select how many quick links/buttons are shown and for each button he could define a target.

Right out of my mind I could imagine that linking the following might make sense:
- dedicated offline list (e.g. owned caches)
- link to an external website (e.g. project-gc.com stating your own profile stats, GC.com owner dashboard, etc pp)
- link to another Android app (e.g. GC Wizard)
-to be continued

**Comment by ztNFny** [comment]
How about gc.com's "newest caches": https://www.geocaching.com/play/search?origin=home&radius=30mi&types=2,3,8,137,5,11,1858,4,9,3773&sort=PlaceDate&asc=False

**Comment by k4ds3** [comment]
> E.g. in https://github.com/cgeo/cgeo/issues/242 it was suggested to add the nearest cache to the dashboard... Or the most recently opened caches?

The nearest caches could imo also be shown as a radar so you instantly see direction & distance information.

**Comment by moving-bits** [comment]
I tend to leave it "as is" - it's a very clean interface, you'll need the space if you have more than one connector enabled, and if the free space looks too boring to you, you can enable displaying the wallpaper. :-)

The only thing that come to my mind as sort of a compromise would be to add four optional icon buttons, which users can select their most favorited actions for (or none, if they don't want to see those buttons). Possible actions could be selected from a range of frequently used actions, some of them mentioned above, mostly those now "hidden" in the home screen's three dot menu.
We could preselect actions that link to faq and manual to make those more prominent to our users.

**Comment by fm-sys** [comment]
We have the "quick launch buttons" meanwhile (https://github.com/cgeo/cgeo/issues/12790#issuecomment-1053657960). If there is anything which should be placed on the dashboard, this can be done as an additional "quick launch button"...

**Comment by moving-bits** [comment]
I propose to close this broad issue here and create new, more specific issues, if required (and leave current home screen as is, if none come up).

**Comment by ztNFny** [comment]
I do have quote some ideas for configurable shortcuts on homescreen:

- Open specific (offline) list (instead of the "lists" view that remembers the last viewed list), either as list or map
- Open list / nearest with filter applied
- Preview specific PQ / bookmark list
